### ISSUE TITLE
textinput element does not have a name available

### ISSUE THREAD

**Issue by libremente** [italia/developers.italia.it#705]
# a11y bug
```
Error: This textinput element does not have a name available to an accessibility API. Valid names are: label element, title undefined, aria-label undefined, aria-labelledby undefined.                                                                                                                                   
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.InputText.Name
   ├── #searchModal > div > div > div:nth-child(2) > div > form > div > div:nth-child(1) > input
   └── <input type="text" class="form-control" placeholder="Search Platforms, software or administrations">
```

**Comment by Trustidiong** [comment]
Hey, I would like to work on this issue. I think it will be fun.

**Comment by mishrajiharsh219** [comment]
Addressing #705
Hey, I would love to work on this, I think it would be fun.

**Comment by bfabio** [comment]
Hey @Trustidiong, are you still working on this?

**Comment by Trustidiong** [comment]
@Fabio I am sorry I had a change in schedule. Please reassign the task to someone else. Thanks.

Sent from Yahoo Mail on Android 
 
  On Thu, Oct 8, 2020 at 6:55 AM, Fabio Bonelli<notifications@github.com> wrote:   


Hey @Trustidiong, are you still working on this?

**Comment by masihtak** [comment]
I can work on this! Do we need labels only forthis field?

**Comment by bfabio** [comment]
Hi @masihtak, we have other [a11y bugs](https://github.com/italia/developers.italia.it/issues?q=is%3Aissue+is%3Aopen+label%3Aa11y) that might be similar.
